balboa-common/src/main/scala/com/socrata/balboa/metrics/MetricIdentifiers.scala

```diff
@@ -7,28 +7,28 @@ import java.util.UUID
  * A way of statically typing some of the entity/metric name keys
  */
 sealed class IdParts(val _parts:Seq[MetricIdPart] = Seq()) {
-  override def toString():String = _parts.mkString("")
-  def getParts = _parts
-  def replacePart(in:MetricIdPart, out:MetricIdPart) = {
+  override def toString: String = _parts.mkString("")
+  def getParts: Seq[MetricIdPart] = _parts
+  def replacePart(in:MetricIdPart, out:MetricIdPart): MetricIdParts = {
     MetricIdParts( getParts.map { p:MetricIdPart => if (p == in) out else p }:_* )  // magic vargs
   }
-  def replaceFirstUnresolved(part:MetricIdPart) = {
-    MetricIdParts( getParts.map { p:MetricIdPart => if (p.isUnresolved()) part else p }:_* )
+  def replaceFirstUnresolved(part:MetricIdPart): MetricIdParts = {
+    MetricIdParts( getParts.map { p:MetricIdPart => if (p.isUnresolved) part else p }:_* )
   }
-  def hasPart(part:MetricIdPart) = {
+  def hasPart(part:MetricIdPart): Boolean = {
     getParts != null && getParts.exists {
       e:MetricIdPart => e == part
     }
   }
-  def isUnresolved() = {
+  def isUnresolved: Boolean = {
```

Comment:
By convention, the () parenthesis in Scala is suppose to mean a method with a side effect. So, things like `close()` come with (), and things like `getX` are without.

I didn't religiously fix up all the files I touched, but if I was in one, I tried to clear additional Intellij warnings. I assume that's why I removed these ones, as Intellij should detect this as a getter.